src/main/java/uk/gov/pay/ledger/performance/dao/PerformanceDao.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.pay.ledger.performance.dao;
+
+import org.jdbi.v3.core.Jdbi;
+import uk.gov.pay.ledger.performance.dao.mapper.PerformanceReportEntityMapper;
+import uk.gov.pay.ledger.performance.entity.PerformanceReportEntity;
+
+import javax.inject.Inject;
+
+public class PerformanceDao {
+
+    private static final String PERFORMANCE_REPORT =
+            "select coalesce(count(amount),0) as volume, coalesce(sum(amount),0) total_amount, " +
+                    "coalesce(avg(amount),0) avg_amount from transaction " +
```

Comment:
[style]
I'd like to see consistency in the query - either use `AS alias` for all of the columns or for none of them (instead of mixing both ways of aliasing columns) - no difference in terms of validity of the query